docs/zh-cn/2.0.2/user_doc/guide/task/map-reduce.md

```diff
@@ -1,4 +1,4 @@
-# MapReduce(MR)节点
+# MapReduce节点
```

Comment:
You should also change version 2.0.3 and dev document

Reply:
ok! I will end at all the needs at the end of February. @zhongjiajie